src/content/blog/astro-5110.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Astro 5.11'
+description: 'Astro 5.11 ships more Content Security Policy (CSP) features, the ability to disable streaming in the Node.js adapter, and more!'
+homepageLink:
+  title: 'Astro 5.11'
+  subtitle: 'Available now!'
+publishDate: '2025-07-02'
+authors:
+  - ema
+coverImage: '/src/content/blog/_images/astro-5110/blog-post-cover-astro-5110.webp'
+socialImage: '/src/content/blog/_images/astro-5110/og-astro-5110.webp'
+lang: 'en'
+---
+
+import Mention from '/src/components/Mention.astro';
+
+**More expermental Content Security Policy goodies, including serving the CSP using your preferred adapter!**
+
+Astro 5.11 is a small minor that ships with new CSP features, and the ability to opt out of HTML streaming when using the Node.js adapter.
+
+- [**CSP with adapters**](#csp-with-adapters)
+- [**Disable HTML streaming**](#disable-html-streaming)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## CSP with adapters
+
+Up until now, static pages could only serve the Content Security Policy (CSP) header via the `<meta http-equiv="content-security-poligy">` element. This was an inconvenience for some users, because some CSP directives such as `report-uri`, `frame-ancestors`, and `sandbox` are not supported in a `<meta>` element.
+
+From Astro 5.11.0, the official `@astrojs/node`, `@astro/netlify` and `@astro/vercel` adapters gained support for specifying custom headers for prerendered pages. This means that these adapters can now serve the CSP header for static pages using the `Response` headers, allowing support for all directives, even on prerendered pages.
+
+This is a new experimental feature available as a new adapter configuration option `experimentalStaticHeaders`:
```

Comment:
Small tweak suggestion to:
- avoid 2x “new”
- avoid “configuration option” as all options configure in a way
- make it a bit more “active” by saying it’s available by “_enabling_” the option

```suggestion
This is an experimental feature available by enabling the new `experimentalStaticHeaders` adapter option:
```